src/store/RelayPendingQueryTracker.js

```diff
@@ -118,12 +118,14 @@ class PendingFetch {
   _errors: Array<?Error>;
 
   constructor(
+    /* eslint-disable comma-dangle */
```

Comment:
It complains about this line?

```
     {pendingFetchMap, preloadQueryMap, storeData}: {
```

Do we need to add [a plugin to strip Flow types](https://www.npmjs.com/package/eslint-plugin-flowtype) before we lint? Know why this might be happening, @zpao?